### PR TITLE
chore: close mcp coverage baseline

### PR DESCRIPTION
**Pull request by jan-kubica**
## Summary

- replace remaining pending MCP dispositions with explicit covered/internal decisions
- remove pending from production handler exposure types and empty the baseline
- add a guard test for the zero-pending baseline state

### CODE REVIEW

File: apps/api/src/handlers/case-law/public-routes.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ import { tSafeId } from "@/api/lib/custom-schema";
 
 const listDecisions = createSafePublicHandler(
   {
-    mcp: { type: "pending" },
+    mcp: { type: "covered", by: "search_case_law" },
```
[the remark this answers is not in the document]
Comment (jan-kubica) [reply apps/api/src/handlers/case-law/public-routes.ts:27]: Accepted and implemented in follow-up PR #1055. The queryless case law browse and facet endpoints are now classified as `internal` with reason `public_indexing`, while `search_case_law` remains attached to the query-bearing search endpoint.

CC on behalf of @jan-kubica
